### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by chasekimball**

### CODE REVIEW

**Comment by chasekimball** [comment]
Hey @bsmithgall! Here's a first attempt at making it pretty, using Skeleton. Thoughts? Am I doing anything silly?

One of the things I'm still flummoxed by, surprise surprise, is vertical alignment. I'd like for the text next to the bubble charts to be aligned in the middle, and for the little colophon at the end to be aligned at the bottom, but I can't seem to get it to work.

File: index.html [hunk 1]
```diff
@@ -2,6 +2,63 @@
...
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/skeleton.css">
+
+  <!-- Favicon
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link rel="icon" type="image/x-icon" href="images/favicon.ico">
+
+
+	<style>
```
Comment (bsmithgall) [inline index.html:30]: I would separate this out into its own file.

File: index.html [hunk 2]
```diff
@@ -2,6 +2,63 @@
...
+  <meta name="description" content="">
+  <meta name="author" content="">
+
+  <!-- Mobile Specific Metas
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- FONT
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link href="http://fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
```
Comment (bsmithgall) [inline index.html:18]: Include this in a separate file

**Review by bsmithgall — COMMENTED** [review]
You should include a screenshot!

**Comment by bsmithgall** [comment]
As for vertical alignment, you have encountered one of my least favorite things in CSS. I refer to [this guide](https://css-tricks.com/centering-css-complete-guide/) all the time.

**Comment by chasekimball** [comment]
I'm going to handle the breaking out of scripts, styles, and data into separate files in a subsequent PR. (See issues https://github.com/chasekimball/fy14-report/issues/10 and https://github.com/chasekimball/fy14-report/issues/4). I'll merge this for now and commit some additional work I've done as well.

**Comment by chasekimball** [comment]
@bsmithgall I haven't done the things you suggested yet, but you can see a live mvp here: https://chasekimball.github.io/fy14-report/ I'm pretty pleased with how it's shaping up. Thanks for your help!

**Pull request merged**